plugins/kotlin/idea/src/org/jetbrains/kotlin/idea/intentions/ConvertTryFinallyToUseCallIntention.kt

```diff
@@ -73,12 +74,14 @@ class ConvertTryFinallyToUseCallIntention : SelfTargetingRangeIntention<KtTryExp
     override fun applicabilityRange(element: KtTryExpression): TextRange? {
         // Single statement in finally, no catch blocks
         val finallySection = element.finallyBlock ?: return null
-        val finallyExpression = finallySection.finalExpression.statements.singleOrNull() ?: return null
+        val finallyExpression = finallySection.finalExpression.statements.singleOrNull()?.let {
+            it.safeAs<KtQualifiedExpression>()?.callExpression ?: it.safeAs<KtCallExpression>()
+        } ?: return null
+        if (finallyExpression.calleeExpression?.text != "close" || finallyExpression.valueArguments.isNotEmpty()) return null
```

Comment:
It would be safer to check `valueArguments` of `resolvedCall`. Otherwise, your code doesn't work for such cases:
```
import java.io.Closeable

class Resource : Closeable {
    fun doStuff(): Unit = println()

    @Deprecated(level = DeprecationLevel.HIDDEN, message = "deprecated")
    override fun close(): Unit = close(1)

    fun close(status: Int = 0): Unit = println(status)
}

fun main() {
    val resource = Resource()
    try {
        resource.doStuff()
    } finally {
        resource.close()
    }
}
```